### ISSUE TITLE
bug: 503: Failed to connect to DB due to PoolTimedOut

### ISSUE THREAD

**Issue by fuyufjh** [fuyufjh/mior#9]
Log:

```
GET /rss?token=xxxx:
   >> Matched: (rss) GET /rss?<token>
   >> Request guard `Connection < Db >` failed: Some(Get(PoolTimedOut)).
   >> Outcome: Failure
   >> No 503 catcher registered. Using Rocket default.
   >> Response succeeded.
```

The cause seems to be https://github.com/launchbadge/sqlx/issues/622. But Rocket 0.5.0-rc.2 still use sqlx 0.5 now, so we may wait for it to update.

**Comment by fuyufjh** [comment]
Still exists...

**Comment by fuyufjh** [comment]
Fixed by 7a60d76341 🎉
